src/reactive_system_urc/include/ReactiveSystemPath.h

```diff
@@ -0,0 +1,111 @@
+/**
+ * Created by William Gu on Nov 3 2018
+ * Declarations for Reactive System Pathfinder
+ * This node is responsible for moving the robot towards a given GPS goal while
+ * avoiding risky areas
+ */
+
+#ifndef PROJECT_ReactiveSystemTwist_H
+#define PROJECT_ReactiveSystemTwist_H
+
+#include <geometry_msgs/Point32.h>
+#include <geometry_msgs/Pose.h>
+#include <geometry_msgs/Twist.h>
+#include <iostream>
+#include <mapping_msgs_urc/RiskArea.h>
+#include <mapping_msgs_urc/RiskAreaArray.h>
+#include <mapping_msgs_urc/RiskAreaStamped.h>
+#include <math.h>
+#include <nav_msgs/Path.h>
+#include <ros/ros.h>
+#include <sb_geom_msgs/Point2D.h>
+#include <sensor_msgs/LaserScan.h>
+#include <std_msgs/Float64.h>
+
+#define MAX_RISK 100
+
+class ReactiveSystemPath {
+  public:
+    /**
+     * Entry point for pathfinding algorithm, produces a path message by
+     * generating multiple possible trajectories in a given range, assigning
+     * each a risk score,
+     * and choosing the trajectory with the lowest risk score
+     * @param risk_areas: risk area ros msg produced by risk assessment system
+     * @param goal_pos: current goal position
+     * @param traj_time_inc: // size of unit increments in a trajectory
+     * calculation
+     * @param traj_num_incs: // number of unit increments in a trajectory
+     * calculation
+     * @param linear_vel: // current linear velocity of robot
+     * @param max_angular_vel: // max turning velocity of robot
+     * @param num_angular_vel: // number of turning velocities to test (each
+     * corresponds to different trajectory)
+     * @param risk_dist_tol_sq: // squared "radius" of robot
```

Comment:
extra `//` at the start of each comment here?

Reply:
I'll get rid of that